Ingredients/Controller/IngredientsController.cs

```diff
@@ -117,4 +117,13 @@ public async Task<IActionResult> AddEdgeFromIngredientToAllergen(string idAllerg
         }
         return Ok();
     }
+    
+    [HttpGet("GetAllIngredients")]
+    public async Task<IActionResult> GetAllIngredients()
+    {
+        var res = await _ingredientsRepository.GetAllIngredients();
+        if (res.Count == 0)
+            return NotFound("There seem to be no entries Ingredients stored here");
```

Comment:
IMO this should not result in a 404. The caller might just expect an empty list instead.